.gitignore

```diff
@@ -92,3 +92,10 @@ tdrs-backend/htmlcov/*
 
 # VIM
 *.swp
+
+# Terraform
+.terraform
+tfapply
+*.tfvars
+*.lock.hcl
+*.tfstate
```

Comment:
`.tfvars` and `.tfstate` often contain secrets and should be ignored. These files and directories are created and used in CI.